Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.cpp

```diff
@@ -197,18 +653,18 @@ void KeyboardMouse::UpdateInput()
   DIMOUSESTATE2 tmp_mouse;
 
   // if mouse position hasn't been updated in a short while, skip a dev state
-  DWORD cur_time = GetTickCount();
-  if (cur_time - m_last_update > DROP_INPUT_TIME)
-  {
-    // set axes to zero
-    m_state_in.mouse = {};
-    m_state_in.relative_mouse = {};
+  // DWORD cur_time = GetTickCount();
```

Comment:
Please don't leave code commented out. If it's not needed, then it should be removed